tests/test_module.py

```diff
@@ -1,25 +1,27 @@
+from typing import Any
+
 import pytest
 
 import zoloto
 
 
-def test_exposes_version():
+def test_exposes_version() -> None:
     assert hasattr(zoloto, "__version__")
 
 
-def test_exposes_marker():
+def test_exposes_marker() -> None:
     assert zoloto.Marker == zoloto.marker.Marker
 
 
-def test_exposes_marker_dict():
+def test_exposes_marker_dict() -> None:
     assert zoloto.MarkerDict == zoloto.marker_dict.MarkerDict
 
 
 @pytest.mark.parametrize(
     "coordinate_struct",
     ["Coordinates", "Orientation", "ThreeDCoordinates", "Spherical"],
 )
-def test_exposes_coordinates(coordinate_struct):
+def test_exposes_coordinates(coordinate_struct: Any) -> None:
```

Comment:
By the looks of the above, this is a string?

```suggestion
def test_exposes_coordinates(coordinate_struct: str) -> None:
```

Reply:
845392c